src/app/components/wrapper-components/part-creation/part-creation.component.html

```diff
@@ -158,44 +158,40 @@ <h1 class="title" i18n>Création d'une partie</h1>
     <div *ngIf="viewInfo.userIsChosenOpponent">
         <div class="message is-primary">
             <div class="message-header">
-                <p i18n>Proposition de configuration</p>
+                <p i18n>Configuration proposal</p>
             </div>
             <div class="message-body">
                 <div *ngIf="viewInfo.creatorIsModifyingConfig else configProposal"
                      class="content">
                     <p i18n>
-                        Vous avez été choisi comme adversaire<br/>
-                        {{ viewInfo.creator }} est en train de modifier la configuration
+                        You have been selected as the opponent.<br/>
+                        {{ viewInfo.creator }} is modifying the configuration.
                     </p>
                 </div>
                 <ng-template #configProposal>
                     <div class="content">
-                        <p>
-                            <ng-container i18n>{{ viewInfo.creator }} propose de faire une partie </ng-container>
-                            <ng-container *ngIf="viewInfo.partType === 'STANDARD'" i18n>standard</ng-container>
-                            <ng-container *ngIf="viewInfo.partType === 'BLITZ'" i18n>rapide</ng-container>
-                            <ng-container *ngIf="viewInfo.partType === 'CUSTOM'" i18n>personnalisée</ng-container>
-                            où :
+                        <p i18n>
+                            {{ viewInfo.creator }} proposes to play a {{ viewInfo.partTypeName }} game, where:
                         </p>
                         <ul>
                             <li>
                                 <span *ngIf="viewInfo.firstPlayer === 'CREATOR'" i18n>
-                                    {{ viewInfo.creator }} joue en premier
+                                    {{ viewInfo.creator }} plays first
                                 </span>
                                 <span *ngIf="viewInfo.firstPlayer === 'CHOSEN_PLAYER'" i18n>
-                                    vous jouez en premier
+                                    you play first
                                 </span>
                                 <span *ngIf="viewInfo.firstPlayer === 'RANDOM'" i18n>
-                                    le premier joueur est tiré au hasard
+                                    the first player is random
```

Comment:
on est sûr que c'est du bon anglais ?

Reply:
bon point, je remplacerais par "is selected randomly"